packages/test-app-frontend/src/App/ITwinJsApp/SavedViewsWidget.tsx

```diff
@@ -67,8 +67,12 @@ export function SavedViewsWidget(props: SavedViewsWidgetProps): ReactElement {
   }
 
   const handleCreateView = async () => {
-    const savedViewData = await captureSavedViewData({ viewport: props.viewport });
-    const savedViewId = await savedViews.actions.submitSavedView("0 Saved View Name", savedViewData);
+    const { viewData, extensions } = await captureSavedViewData({ viewport: props.viewport });
```

Comment:
How does a consumer pass their own extension "capture" logic if they want to to make the widget store custom extensions?

Reply:
I need to do a separate PR that updates and adds new usage examples.

Hypothetically consumer's code would already look like

```ts
async function createSavedView(viewport) {
  const { viewData, extensions } = await captureSavedViewData({ viewport });
  const thumbnail = captureSavedViewThumbnail(viewport);
  await publishSavedView({ viewData, extensions, thumbnail });
}
```

All they would have to do is

```ts
  await publishSavedView({
    viewData,
    extensions: extensions.concat(captureCustomExtensions(viewport)),
    thumbnail,
  });
```